index.html

```diff
@@ -53,6 +53,149 @@ <h1  class="headline1">I'm Loukmane
+            <p>javaScript</p>
+          </div>
+          <div class="view-project">
+          <p>See Project</p>
+        </div>
+          </li>
+          <li class="work work2">
+            <img class="img1" src="portfolioimgs/Snap2.png" alt="work1Snapshot">
+              <h1 class="multi">Multi-Post
+                Stories</h1>
+              <div class="canopy">
+              <h3>CANOPY</h3>
+              <ul class="cards">
+                <li class="info">Back End Dev</li>
+                <li class="info">2015</li></ul>
+              </div>
+              <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+             <div class="tools"> <p>html</p>
+              <p>css</p>
+              <p>javaScript</p>
+            </div>
+            <div class="view-project">
+            <p>See Project</p>
+          </div>
+            </li>
+            <li class="work work3">
+              <img class="img1" src="portfolioimgs/Snap3.png" alt="work1Snapshot">
+                <h1>Tonic</h1>
+                <div class="canopy">
+                <h3>CANOPY</h3>
+                <ul class="cards">
+                  <li class="info">Back End Dev</li>
+                  <li class="info">2015</li></ul>
+                </div>
+                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+               <div class="tools"> <p>html</p>
+                <p>css</p>
+                <p>javaScript</p>
+              </div>
+              <div class="view-project">
+              <p>See Project</p>
+            </div>
+              </li>
+              <li class="work work4">
+                <img class="img1" src="portfolioimgs/Snap4.png" alt="work1Snapshot">
+                  <h1 class="multi">Multi-Post
+                    Stories</h1>
+                  <div class="canopy">
+                  <h3>CANOPY</h3>
+                  <ul class="cards">
+                    <li class="info">Back End Dev</li>
+                    <li class="info">2015</li></ul>
+                  </div>
+                  <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                 <div class="tools"> <p>html</p>
+                  <p>css</p>
+                  <p>javaScript</p>
+                </div>
+                <div class="view-project">
+                <p>See Project</p>
+              </div>
+                </li>
+         </ul>
+       </section>
+
+       <section class="about-section">
+        <div class="about">
+         <h1 class="aboutMySelf" >About<br>Myself</h1>
+          <p class="text text2 ">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+<p class="contact">LET'S CONNECT</p>
```

Comment:
- [ ] Please make sure to use a `hx` tag instead of a `p`, doing this will make your code more semantic. 

Semantic HTML is the key starting point toward good accessibility practices. Many screen readers have functionality that allows a user to select to read only the headings on the page or only the links. :computer: